prov/util/src/util_mem_monitor.c

```diff
@@ -121,12 +123,17 @@ void ofi_monitors_init(void)
 	fi_param_define(NULL, "mr_cuda_cache_monitor_enabled", FI_PARAM_BOOL,
 			"Enable or disable the CUDA cache memory monitor."
 			"Monitor is enabled by default.");
+	fi_param_define(NULL, "mr_rocr_cache_monitor_enabled", FI_PARAM_BOOL,
+			"Enable or disable the ROCR cache memory monitor. "
+			"Monitor is enabled by default.");
```

Comment:
Same here.  We need some standard mechanism to define the parameters and read them for each monitor.